src/status.rs

```diff
@@ -225,6 +222,136 @@ pub enum StatusCode {
 
 impl StatusCode {
 
+    #[doc(hidden)]
```

Comment:
The only reason I can see for using `from_u16` is to handle parsing, which we already do. If you know the number yourself, just write the variant instead.